config/optional/jsonapi_extras.jsonapi_resource_config.paragraph--event_details.yml

```diff
@@ -0,0 +1,150 @@
+langcode: en
```

Comment:
A reference to this file should be put in the (info file here)[https://github.com/dpc-sdp/tide_event/blob/d50a82591fdb7c88dc90d04b37ad125ffad70a4d/tide_event.info.yml#L14]. This helps when exporting config.

Reply:
Good catch 👍